syncd/VirtualOidTranslator.cpp

```diff
@@ -22,6 +22,28 @@ VirtualOidTranslator::VirtualOidTranslator(
     // empty
 }
 
+bool VirtualOidTranslator::tryTranslateRidToVid(
+        _In_ sai_object_id_t rid,
+        _Out_ sai_object_id_t &vid)
+{
+    SWSS_LOG_ENTER();
+
+    std::lock_guard<std::mutex> lock(m_mutex);
+
+    if (rid == SAI_NULL_OBJECT_ID)
+        return false;
```

Comment:
NULL value should always be translated to NULL, and this is perfect scenario, event checkRidExists return true in that case

Reply:
ok